DuckDuckGo/Subscription/Views/SubscriptionITPView.swift

```diff
@@ -163,4 +163,12 @@ struct SubscriptionITPView: View {
         navAppearance.tintColor = UIColor(designSystemColor: .textPrimary)
     }
 }
+
+// @available(iOS 15.0, *)
```

Comment:
Same as preview comments above

Reply:
Explanation added and description updated in https://app.asana.com/0/414709148257752/1206774081310425/f